templates/plugin/turbostat.conf.epp

```diff
@@ -0,0 +1,25 @@
+<Plugin turbostat>
+<% if $::collectd::plugin::turbostat::core_c_states { -%>
```

Comment:
Drop the `$::` - it's not needed

Reply:
I have dropped the useless '::', but the '$' is required.